src/DonorProfiles/resources/js/app/components/select-control/index.js

```diff
@@ -0,0 +1,103 @@
+// Import vendor dependencies
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+// Import utilities
+import { toKebabCase } from '../../utils';
+
+// Import styles
+import './style.scss';
+
+const { __ } = wp.i18n;
+
+const SelectControl = ( { label, value, isLoading, onChange, options, placeholder, width } ) => {
+	if ( options && options.length < 2 ) {
+		return null;
+	}
+
+	const selectedOptionValue = options !== null ? options.filter( option => option.value === value ) : null;
+	const selectStyles = {
```

Comment:
What is the reason that selected styles are defined in JS as opposed to in the stylesheet?

Reply:
In this case, the styles object is passed to the Select component as a first-class prop which allows us to conditionally render styles based on the select component state. Additionally, because the select component uses dynamically generated classes it is difficult to effectively target elements it renders. Tldr: we do this because it is the officially supported method of styling Select components rendered by `react-select`.

Reply:
Sounds good to me!